### PR TITLE
[facebox] initial release

### PR DESCRIPTION
**Pull request by runningman84**
**Description of the change**
This chart can be used with double-take like deepstack.

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

File: charts/incubator/facebox/ci/ct-values.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+env:
+  TZ: "Africa/Cairo"
+  LATITUDE: "29.9792"
+  LONGITUDE: "31.1342"
+  ELEVATION: "64"
+  DASH_URL: "http://0.0.0.0:5050"
+ingress:
+  main:
+    enabled: true
```
Comment (onedr0p) [inline charts/incubator/facebox/ci/ct-values.yaml:9]: This looks like a bad copy/paste?

**Review by onedr0p — CHANGES_REQUESTED** [review]
I think you can sldo put this in the stable directory, not incubator.

File: charts/incubator/facebox/ci/ct-values.yaml (hunk 1 above)
Comment (runningman84) [reply charts/incubator/facebox/ci/ct-values.yaml]: yes I fixed that...

**Comment by onedr0p** [comment]
```
[INFO]     facebox v1
:-(
	Ohh you're so close, but Machine Box requires your Box Key in order to work.

	Head over to machinebox.io/account to get your key, and use it
	as the value for the MB_KEY environment variable.

	If you get stuck, read our guide: machinebox.io/docs/setup/box-key

	See you on machinebox.io/account :)```

**Comment by onedr0p** [comment]
You might need to exclude this chart from being installed as a tes, please add it to the list here in your PR

https://github.com/k8s-at-home/charts/blob/master/.github/ct-install.yaml#L7-L19

File: charts/stable/facebox/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+sources:
+- https://docs.veritone.com/#/developer/machine-box/boxes/facebox-overview
+- https://hub.docker.com/r/machinebox/facebox
+maintainers:
+- name: philipp
+  email: phil@hellmi.de
+dependencies:
+- name: common
+  repository: https://library-charts.k8s-at-home.com
+  version: 4.2.0
```
Comment (bjw-s) [inline charts/stable/facebox/Chart.yaml:21]:
```suggestion
  version: 4.3.0
```

**Review by bjw-s — APPROVED** [review]

**Pull request merged**